playground/mocks/data/idp/idx/authenticator-expired-custom-password.json

```diff
@@ -0,0 +1,179 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "02.id.0I3VrWZIsEIsQrYO17rdL-P79gY9SdXsAXIQYDqN~c.KnKsrI0K_y3oxX5zulVxiGvUfKPUx14nHDe4K82BT7FA0Fy6fc2VXetiA_B4ql6FyC70EhWaGQRiwvmpweITu6wWqnHbqXhFLt48jEWrml9iGo_wHnm6OyTpzSwpy8ZHPiCMQXHmjjFqyvJnnyd1fpSvpJRJx_MTq4wh54lxMa8NWyL8W0uG0k-Vi2nvvdplAqCEUANlubObELVejUQLqd0vVKOZKNwFufo51h8EYfUwXDX4pahM4mPIugZw9Iw6vx_eKIXo3feCc2aGuLo-BtyUod_QP-6iQingpEJ2XnRlPy9nfTbOWn5dlepbOT3CqsZXtysBbFOjns21Y1KhLc_3X2CB-DUmEj1eZI4fyemmvPklNKJER-lyy2maI1s8LcmugXWluUo-ZlfCx8ZVzkRLPJBA1lcHxva5awKJWUS4XeIXUMC_pbqLfL02GsKk6dRJPM1Ses8UH9XJeuEW1mSsjLnuIh-eVeZkFZwmzwLyJpx5sglfVkcHxi9JYPhM5N2YJEjM_wnYFEce7HLjQRMiJ_avpqAhcKUBzU9s4h8MiHp9GXSmPigasK67huifKsGahM6pTEj2rLYbIZcl63ysjnhDdk-M9TdGKipCKd7-kemBGrducUtTee-jokq10bcEZI9scfLs8knb_ZWMvanFJiovXlhVFjPRga9ViDiztVvaANLqP7YECE-94KyWVeQkMOprzzZy2a5p5UzEYI_HGKyfclVtTpsaWz7rpp3EDgtcP8iFQ14BuXh1nFZUA6rttvyTl5UMzCaA5jK4C48qB4sXDqzF4zjmqO93nbZPVQTGElN7gxuYAtHxYKAzbomRelUbVctfr599P73KTC1TFgFU1KRZW-2NzAnu9dgBs2KbWzdZ1S02CwbxUDF-rQ0WN5ynfKVa6GyIS3X4W8h_ha-8vKzl2ZeEGj8nxSvNB32-tU7EXDtaARoH-2Pxa3CCxm5r_L-CTayP_zytN7yHMEkzDSpriFHlMajyWIRVYCLf8Gm76475L8om7TQrje6Gno8wkdHdKGttRyqCwepGqzb9AKL2mJInkSAQo0m9Ekpf1DZs9bcN241HExuHD90PtQvXDlNQuAIiwW_3-uMMuT3vshk1zVdChLlUiGOr6_sUXO9vYCXDeXCh3UFz",
+  "expiresAt": "2022-11-28T19:11:55.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": ["create-form"],
+        "name": "reenroll-custom-authenticator",
+        "relatesTo": ["$.currentAuthenticator"],
+        "href": "https://sun.okta1.com/idp/idx/challenge/answer",
+        "method": "POST",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "value": [
+          {
+            "name": "credentials",
+            "type": "object",
+            "form": {
+              "value": [
+                { "name": "passcode", "label": "New password", "secret": true },
+                {
+                  "name": "revokeSessions",
```

Comment:
Does this change depend on https://github.com/okta/okta-signin-widget/pull/2896?

Reply:
it does not, this is a separate feature